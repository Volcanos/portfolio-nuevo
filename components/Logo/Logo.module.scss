.logo {
  font-size: 1.6rem;
  margin-bottom: 2rem;
  font-family: 'Advent Pro', sans-serif;
  font-weight: 500;
}
