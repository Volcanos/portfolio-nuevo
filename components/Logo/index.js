import styles from './Logo.module.scss';

const Logo = () => {
    return (
        <div className={styles.logo}>Borja Gracia Mart√≠n</div>
    )
}

export default Logo;