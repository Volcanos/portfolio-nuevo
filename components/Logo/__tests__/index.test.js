import React from 'react'
import { mount, render } from 'enzyme';
import 'jest-styled-components'
import Logo from '../index';

describe('Logo', function () {
    it('should render to static and contain Borja Gracia Martín', function () {
        const text = render(<Logo />).text();
        expect(text).toEqual('Borja Gracia Martín');
    });

    it('snapshot not change', function () {
        const tree = render(<Logo />);
        expect(tree).toMatchSnapshot();
    });
});