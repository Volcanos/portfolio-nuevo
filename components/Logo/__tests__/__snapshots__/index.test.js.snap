// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo snapshot not change 1`] = `
.c0 {
  font-size: 1.6rem;
  margin-bottom: 2rem;
}

<h1
  class="c0"
>
  Borja Gracia Mart√≠n
</h1>
`;
